packages/edit-site/src/components/sidebar-navigation-screen/style.scss

```diff
@@ -18,7 +18,7 @@
 .edit-site-sidebar-navigation-screen__content {
 	padding: 0 $grid-unit-20;
 
-	.edit-site-sidebar-navigation-screen-details-footer {
+	.components-item-group {
```

Comment:
An alternative to targeting the inner component's class name, which could change without notice, would be to add a className to the `<ItemGroup />` here:

https://github.com/WordPress/gutenberg/blob/6e884dcc29f4a3036ec1446bc13b256536e63147/packages/edit-site/src/components/sidebar-navigation-screen-main/index.js#L26-L26

Reply:
Yes but then it should be done for all the `<ItemGroup>`s in all the navigation sub-panels. Not sure it's in the scope of this PR though a follow-up for a better solution would be welcome.

Reply:
I recommend avoiding selectors with the `components-` prefix whenever possible, otherwise this style may have unintended side effects if an ItemGroup component is added to the sidebar in the future.

This is the only place in all of Gutenberg where the `.components-item-group` selector is used. Admittedly, there are many other `ItemGroup` components that don't have CSS class names, but since they don't have their own styling, I don't think they need CSS class names in the first place.

Reply:
@t-hamano we all agree CSS selectors from components should be avoided. However, this PR scope is to just fix a regression by restoring the previous usage where `.components-item-group` was already in use. We should avoid scope creep. Further improvements can be made in a follow-up.

Reply:
Since what I proposed is just a small code quality improvement and it targets the same component, I think it's okay to include it in this PR.

Of course, we can address it in a follow-up, so feel free to merge it as is.